File: C/Estrutura condicional/multiplos.c
// Biblioeteca(s)
#include <stdio.h>

int main()
{
    // Variáveis
    int x, y, novo;


    // Entrada de dados
    printf("Digite o primeiro numero: ");
    scanf("%d", &x);

    printf("Digite o segundo numero: ");
    scanf("%d", &y);

    // Condicional
    if (x % y == 0 || y % x == 0) {
        printf("Sao multiplos!");
    }
    else {
        printf("Nao sao multiplos");
    }

    return 0;
}

File: C#/Estrutura condicional/multiplos.cs
// Variáveis
int x, y;

// Entrada de dados
Console.Write("Digite o primeiro número: ");
x = int.Parse(Console.ReadLine());

Console.Write("Digite o segundo número: ");
y = int.Parse(Console.ReadLine());

// Condicional
if (x % y == 0 || y % x == 0)
{
    Console.Write("Eles são múltiplos!");
}

else
{
    Console.Write("Eles NÃO são múltiplos.");
}